### PR TITLE
Collected suggestions from http://translator.tosbase.com/.

### PR DESCRIPTION
**Pull request by TOSBase**
Collected suggestions from http://translator.tosbase.com/.
Contributors: http://translator.tosbase.com/contributors/

I used a new method that prevented some conflicts so I hope everything is fine this time.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -10456,10 +10456,10 @@ ETC_20150428_011434	Recipe - Superior Rune Bracelet
...
-ETC_20150428_011439	FAQ: Quick Slot{nl}1) You can add items or skills to the quick slot on the lower part of your skill. {nl}2) Skills are automatically added to quick slot when you learn quests. {nl}3)You can use up to 40 quick slots. {nl}4) Quick slots can be locked or unlocked. {nl}You cannot add or remove items or skill in quick slot when it is locked.
+ETC_20150428_011438	FAQ: Character Level and Class Level {nl}You can check the info at the lower part of the screen. {nl}2) Character Level: this is your character's level. {nl}3) Class Level: this is the level of the class you chose. {nl} You can advance to another class when you reach level 15.
+ETC_20150428_011439	FAQ: Quick Slot{nl}1) You can add items or skills to the quick slot on the lower part of your skill. {nl}2) Skills are automatically added to quick slot when you learn skills. {nl}3)You can use up to 40 quick slots. {nl}4) Quick slots can be locked or unlocked. {nl}You cannot add or remove items or skill in quick slot when it is locked.
```
Comment (Chaosbelow) [inline ETC.tsv]: Change "on the lower part of your skill" to " at the lower part of your screen."

File: ETC.tsv [hunk 2]
```diff
@@ -10534,12 +10534,12 @@ ETC_20150428_011513	You need to kill the monsters to open the box!
...
 ETC_20150428_011515	Activating the Altar
 ETC_20150428_011516	Unclean Insect Eggs
-ETC_20150428_011517	It can only be used in the Tomb Building Chapel.
+ETC_20150428_011517	It can only be used inside the Tomb Building Chapel.
 ETC_20150428_011518	Infrobirk Flame
 ETC_20150428_011519	Absorbed all the magical power.
 ETC_20150428_011520	Activate the Chapel Altar to kill the monsters!
 ETC_20150428_011521	I will open the door
-ETC_20150428_011522	Attacks are impossible because of the magic barrier
+ETC_20150428_011522	Attacks are ineffective because of the magic barrier
```
Comment (Chaosbelow) [inline ETC.tsv]: Ineffective would mean the attacks are hitting but are not having a desired effect. Impossible is implying that attacks are not possible at all. Two different meanings, leave original translation

**Comment by Chaosbelow** [comment]
Good job, thank you :+1:

File: ITEM.tsv [hunk 3]
```diff
@@ -653,7 +653,7 @@ ITEM_20150317_000652	The recipe for this shield has only been discovered recentl
 ITEM_20150317_000653	Meteor
 ITEM_20150317_000654	 - 1% chance to regenerate 200 HP every second for 5 seconds when hit
 ITEM_20150317_000655	Sage Wall
-ITEM_20150317_000656	There are records of Mayburn using this kind of shield when he used to adventure around the world prior to his dwelling in the Cathedral.
+ITEM_20150317_000656	There are records of Mayburn using this kind of shield when he used to travel around the world prior to his dwelling in the Cathedral.
```
Comment (Chaosbelow) [inline ITEM.tsv]: Unnecessary change, use original translation

**Comment by Sunaries** [comment]
The changes made to ITEM.tsv are incorrect. The names of equipment have already been translated in the past. Now they're missing again... Use this for the equipment names: https://github.com/Treeofsavior/EnglishTranslation/wiki/2.3.%20Equipment

File: ITEM.tsv [hunk 4]
```diff
@@ -8,14 +8,14 @@ ITEM_20150317_000007	Long Armband
...
 ITEM_20150317_000009	Tasseled Armband
 ITEM_20150317_000010	Empty_weapon
-ITEM_20150317_000011	
+ITEM_20150317_000011	Coarse Gladius
 ITEM_20150317_000012	One-Handed Sword [Slash]
 ITEM_20150317_000013	A weapon that is commonly used and carried by soldiers of the kingdom.
 ITEM_20150317_000014	Gladius
 ITEM_20150317_000015	This weapon is crafted with room for improvement over its military version.
-ITEM_20150317_000016	
+ITEM_20150317_000016	Low Polchyeon
```
Comment (Sunaries) [inline ITEM.tsv]: Low Falchion

File: ITEM.tsv [hunk 5]
```diff
@@ -8,14 +8,14 @@ ITEM_20150317_000007	Long Armband
...
 ITEM_20150317_000017	This sword is exceptional for slashing.
-ITEM_20150317_000018	
+ITEM_20150317_000018	Cass's Low Karas
```
Comment (Sunaries) [inline ITEM.tsv]: Low Kaskara

File: ITEM.tsv [hunk 6]
```diff
@@ -29,22 +29,22 @@ ITEM_20150317_000028	Kris
 ITEM_20150317_000029	Both practical and decorative, the dagger's curved edge amplifies the damage it inflicts.
 ITEM_20150317_000030	Snickersnee
 ITEM_20150317_000031	Its extremely sharp blade makes it a favorite even among the most experienced swordsmen.
-ITEM_20150317_000032	
-ITEM_20150317_000033	
+ITEM_20150317_000032	Coarse Siamese Cyr
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Shamshir

File: ITEM.tsv [hunk 7]
```diff
@@ -29,22 +29,22 @@ ITEM_20150317_000028	Kris
...
+ITEM_20150317_000033	Coarse Scimitar
 ITEM_20150317_000034	Soldier's Gladius
 ITEM_20150317_000035	Basic armament provided to full-time soldiers. Its overall quality is better than the ones sold in the market.
 ITEM_20150317_000036	Mayor's Falchion
 ITEM_20150317_000037	Looking at this weapon raises suspicions on the previous owner's past.
-ITEM_20150317_000038	
+ITEM_20150317_000038	Crude Castile Karas
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Kaskara

File: ITEM.tsv [hunk 8]
```diff
@@ -29,22 +29,22 @@ ITEM_20150317_000028	Kris
...
 ITEM_20150317_000039	Moplah
 ITEM_20150317_000040	This sword combines both practicality and aesthetics.
-ITEM_20150317_000041	
+ITEM_20150317_000041	Heavy Gladius
 ITEM_20150317_000042	Falchion
-ITEM_20150317_000043	
+ITEM_20150317_000043	Heavy Polchyeon
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Falchion

File: ITEM.tsv [hunk 9]
```diff
@@ -29,22 +29,22 @@ ITEM_20150317_000028	Kris
...
 ITEM_20150317_000044	Kaskara
-ITEM_20150317_000045	
-ITEM_20150317_000046	
-ITEM_20150317_000047	
+ITEM_20150317_000045	Heavy Cinque Oradea
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Cinquedea

File: ITEM.tsv [hunk 10]
```diff
@@ -29,22 +29,22 @@ ITEM_20150317_000028	Kris
...
+ITEM_20150317_000046	Heavy Sabre
+ITEM_20150317_000047	Heavy Siamese Cyr
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Shamshir

File: ITEM.tsv [hunk 11]
```diff
@@ -136,14 +136,14 @@ ITEM_20150317_000135	Quaddara
 ITEM_20150317_000136	Contrary to its appearance, this sword is double-edged and often used by swordsmen pursuing practicality.
 ITEM_20150317_000137	Light Claymore
 ITEM_20150317_000138	The shape of the blade enables you to perform a strikedown attack.
-ITEM_20150317_000139	
+ITEM_20150317_000139	Fortified Coliseum
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Colossus

File: ITEM.tsv [hunk 12]
```diff
@@ -136,14 +136,14 @@ ITEM_20150317_000135	Quaddara
...
 ITEM_20150317_000140	Eillun
 ITEM_20150317_000141	Initially even the Great Ruklyss carried this sword.
 ITEM_20150317_000142	Practice Bastard Sword / Training Bastard Sword
 ITEM_20150317_000143	Bastard Sword
-ITEM_20150317_000144	
-ITEM_20150317_000145	
-ITEM_20150317_000146	
+ITEM_20150317_000144	Grand Juba's City
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Flamberge

File: ITEM.tsv [hunk 13]
```diff
@@ -136,14 +136,14 @@ ITEM_20150317_000135	Quaddara
...
+ITEM_20150317_000145	Great Value Platform
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Nodachi

File: ITEM.tsv [hunk 14]
```diff
@@ -205,14 +205,14 @@ ITEM_20150317_000204	Nulis
 ITEM_20150317_000205	Extremely efficient against legendary beings who do not exist in the normal realm of night and day.
 ITEM_20150317_000206	Gravity
 ITEM_20150317_000207	Gravity cannot be avoided, whomever or wherever you are. But if you adapt to it, no one can underestimate such power.
-ITEM_20150317_000208	
+ITEM_20150317_000208	Evilish Road
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Short Rod

File: ITEM.tsv [hunk 15]
```diff
@@ -205,14 +205,14 @@ ITEM_20150317_000204	Nulis
...
 ITEM_20150317_000209	Rod [Strike]
 ITEM_20150317_000210	Anyone who has a plan to study magic arts end up with this sooner or later.
 ITEM_20150317_000211	Short Rod
 ITEM_20150317_000212	When young Magicians graduate from Apprentice-hood, they are given this weapon.
-ITEM_20150317_000213	
+ITEM_20150317_000213	Dull Road
```
Comment (Sunaries) [inline ITEM.tsv]: Low Long Rod

File: ITEM.tsv [hunk 16]
```diff
@@ -205,14 +205,14 @@ ITEM_20150317_000204	Nulis
...
 ITEM_20150317_000214	The decorations found in mage's equipment are added for function, not only for good appearance.
-ITEM_20150317_000215	
+ITEM_20150317_000215	Compact Rover
```
Comment (Sunaries) [inline ITEM.tsv]: Low Cane Rover

File: ITEM.tsv [hunk 17]
```diff
@@ -226,16 +226,16 @@ ITEM_20150317_000225	Rune Rod
 ITEM_20150317_000226	A Magical Rune is used to amplify the power of the rod.
 ITEM_20150317_000227	Crystal Rod
 ITEM_20150317_000228	A precise fitting of the jewel and the shaft is required to craft this rod.
-ITEM_20150317_000229	
-ITEM_20150317_000230	
+ITEM_20150317_000229	Worn Computer
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Puter Rod

File: ITEM.tsv [hunk 18]
```diff
@@ -226,16 +226,16 @@ ITEM_20150317_000225	Rune Rod
...
+ITEM_20150317_000230	Old Owl
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Owl Rod

File: ITEM.tsv [hunk 19]
```diff
@@ -226,16 +226,16 @@ ITEM_20150317_000225	Rune Rod
...
 ITEM_20150317_000231	Soldier's Short Rod
 ITEM_20150317_000232	This rod would be given to magicians who joined the army. Though the procedure has been abolished since the Great Plant Cataclysm, abundant stock still remains.
 ITEM_20150317_000233	Chieftain's Long Rod
 ITEM_20150317_000234	The shape of this rod was popular in the past. Although old-fashioned, the rod performs rather decently.
-ITEM_20150317_000235	
+ITEM_20150317_000235	Very Compact Rover
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Cane Rover

File: ITEM.tsv [hunk 20]
```diff
@@ -226,16 +226,16 @@ ITEM_20150317_000225	Rune Rod
...
 ITEM_20150317_000236	Battle Rod
 ITEM_20150317_000237	Besides of assisting the magic casting, this rod was also designed for physical combat.
-ITEM_20150317_000238	
+ITEM_20150317_000238	Low Explosive Bullets
```
Comment (Sunaries) [inline ITEM.tsv]: Heavy Short Rod

File: ITEM.tsv [hunk 21]
```diff
@@ -323,7 +323,7 @@ ITEM_20150317_000322	Gulyle
 ITEM_20150317_000323	The first ever recorded case of applying magic in the bowcrafting process. Since then it has found its own market.
 ITEM_20150317_000324	Karman
 ITEM_20150317_000325	A bow modified for more power by archers confident in their arm strength.
-ITEM_20150317_000326	
+ITEM_20150317_000326	Roca's Crude Bow
```
Comment (Sunaries) [inline ITEM.tsv]: Coarse Rokas Bow

File: ETC.tsv [hunk 22]
```diff
@@ -7451,7 +7451,7 @@ ETC_20150317_008011	N/A
 ETC_20150317_008012	N/A
 ETC_20150317_008013	N/A
 ETC_20150317_008014	N/A
-ETC_20150317_008015	N/A
+ETC_20150317_008015	The minimum number of combos
```
Comment (Chaosbelow) [inline ETC.tsv]: N/A lines are not to be translated. Reference: pull #445 and issue #449

File: ETC.tsv [hunk 23]
```diff
@@ -7460,8 +7460,8 @@ ETC_20150317_008020	N/A
 ETC_20150317_008021	N/A
 ETC_20150317_008022	N/A
 ETC_20150317_008023	N/A
-ETC_20150317_008024	N/A
-ETC_20150317_008025	N/A
+ETC_20150317_008024	The maximum number of options
```
Comment (Chaosbelow) [inline ETC.tsv]: N/A lines are not to be translated. Reference: pull #445 and issue #449

File: ETC.tsv [hunk 24]
```diff
@@ -7460,8 +7460,8 @@ ETC_20150317_008020	N/A
...
+ETC_20150317_008025	The maximum number of sockets
```
Comment (Chaosbelow) [inline ETC.tsv]: N/A lines are not to be translated. Reference: pull #445 and issue #449

**Comment by imcgames** [comment]
Thank you :+1: 

(Do the list of contributors in the link have github accounts too? If so, I'd like to recommend their names to match the github accounts :wink: )
